systemtests/aci_test.go

```diff
@@ -59,3 +59,48 @@ func (s *systemtestSuite) TestACIMode(c *C) {
 	c.Assert(s.cli.EndpointGroupDelete("default", "epgB"), IsNil)
 	c.Assert(s.cli.NetworkDelete("default", "aciNet"), IsNil)
 }
+
```

Comment:
Would be good to move this to a separate file which can hold all tests that require ACI physical fabric.